docs/staff/surveys.md

```
@@ -0,0 +1,387 @@
+
+### Survey Assignment
+
+Control which students can access and respond to your survey:
+
+**Assign to all students** (default)
+
+- All enrolled students in the course can see and respond to the survey
+- Response rate is calculated based on total enrolled students
+- Best for course-wide feedback or evaluations
+
+**Assign to specific students**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)